data/service/raw-data-service.js

```diff
@@ -435,6 +435,12 @@ exports.RawDataService = DataService.specialize(/** @lends RawDataService.protot
             }
             this._addMapDataPromiseForStream(result, stream);
 
+
+            //TO-DO: #warning
```

Comment:
I think we use `TODO` everywhere else. Many editors have automatic support for finding TODO comments in code.

Reply:
taken care of